com.unity.cinemachine/Documentation~/CinemachineUsing.md

```diff
@@ -35,7 +35,7 @@ The Cinemachine Brain is a component in the Unity Camera itself. The Cinemachine
 
 ## Positioning and Aiming
 
-Use the [__Position Control__ properties](CinemachineVirtualCameraBody.md) in a CmCamera to specify how to move it in the Scene. Use the [__Rotation Control__ properties](CinemachineVirtualCameraAim.md) to specify how to aim it.
+Use the [__Position Control__ properties](CmCamera.md) in a CmCamera to specify how to move it in the Scene. Use the [__Rotation Control__ properties](CinemachineVirtualCameraAim.md) to specify how to aim it.
```

Comment:
Not sure this is the right hotlink.  It should take you somewhere analogous to the second one in the line